cx_Freeze/freezer.py

```diff
@@ -572,7 +572,9 @@ def _WriteModules(self, fileName, finder):
 
         if not self.silent:
             self._PrintReport(fileName, modules)
-        finder.ReportMissingModules()
+
+        if not self.silent:
```

Comment:
An indentation is sufficient to place it in the silent block. But, I wonder why the original author did not do this.

Reply:
I know. I put it in a separate block as a hint to people in the future that they may want to consider different conditions for calling the ReportMissingModules() method.

I guess that the original author probably did not make the method conditional on "no self.self", because he viewed the missing modules as an error that the user should be informed about in any case. However, the problem is that ReportMissingModules() reports many "missing" modules no matter what-- for example, platform specific modules that are not available on other modules.

We could put in a safe list, of modules that we know are okay to be missing, and ignore those ones.  But that seems a lot more complicted.

Reply:
> reports many "missing" modules no matter what

It also bothers me and I've tried to reduce it, because sometimes it helps to identify what is wrong, but with a very long list, it doesn't help at all ...

Reply:
Yeah. 

I think if someone knows enough to be helped by the list of missing modules, then they would also know enough to turn off "silent" when they encounter a problem.

Reply:
Incidentally, I was looking at distutils.  The way it handles the verbosity is by setting a global reporting threshold in the distutils.log module, and directing all the output through distutils.log.log() (and some sibling functions).  Should cx_Freeze standardize on doing that as well?

Reply:
Are you okay having ReportMissingModules() silenced?  If you still want ReportMissingModules() to run by default, we could add another option specifically for it ("--hide-missing"?).